ignite/pkg/protoanalysis/protoutil/helpers.go

```diff
@@ -248,6 +248,39 @@ func GetImportByPath(f *proto.Proto, path string) (node *proto.Import, err error
 	return nil, errors.Errorf("import %s not found", path)
 }
 
+// GetFieldByName returns the field with the given name or nil if not found within a message.
+// Only traverses in proto.Message since they are the only nodes that contain fields:
+//
+//		f, _ := ParseProtoPath("foo.proto")
+//		m := GetMessageByName(f, "Foo")
+//		f := GetFieldByName(m, "Bar")
+//	 f.Name // "Bar"
+func GetFieldByName(f *proto.Message, name string) (node *proto.NormalField, err error) {
+	node, err = nil, nil
```

Comment:
```suggestion
```

Reply:
I think we can skip using naked return here since we need to set it to nil. We can use this line to declare the var

```suggestion
	var (
		node *proto.NormalField
		err error
	)
```

Reply:
True, I simplify kept the same style as the functions above, but I'll simplify here and there as well.